src/test/java/plugins/WorkflowPluginTest.java

```diff
@@ -243,7 +248,7 @@
             "}");
         job.sandbox.check();
         job.save();
-        assertThat(job.startBuild().shouldSucceed().getConsole(), containsString("-> FETCH_HEAD"));
+        assertThat(job.startBuild().shouldSucceed().getConsole(), containsString("-&gt; FETCH_HEAD"));
```

Comment:
@Vlatombe it seems that #278 was an incompatible change? This change should not be necessary.